ui/pages/home/home.component.js

```diff
@@ -330,6 +334,29 @@ export default class Home extends PureComponent {
             }
           />
         ) : null}
+
+        {removeCollectibleMessage === 'success' ? (
```

Comment:
Optional: you can refactor as 
```
{removeCollectibleMessage === 'success' && <ActionableMessage />}
```

Reply:
We can refactor it to use `&&` but I believe we should prefer using ternary operators here.

Reply:
I don't have a strong opinion on this, we'll go for whichever one you are more comfortable with 🙏